scylla/src/transport/session_test.rs

```diff
@@ -33,6 +38,38 @@ use std::time::Duration;
 use tokio::net::TcpListener;
 use uuid::Uuid;
 
+#[cfg(not(feature = "cloud"))]
+type ConditionalSessionBuilderKind = DefaultMode;
+#[cfg(feature = "cloud")]
+type ConditionalSessionBuilderKind = CloudMode;
+
+/*
+  Creates a generic session builder based on conditional compilation configuration
+  For SessionBuilder of DefaultMode type, adds localhost to known hosts, as all of the tests
+  connect to localhost.
+*/
+fn create_new_session_builder() -> GenericSessionBuilder<ConditionalSessionBuilderKind> {
+    #[cfg(not(feature = "cloud"))]
+    {
+        let uri = std::env::var("SCYLLA_URI").unwrap_or_else(|_| "127.0.0.1:9042".to_string());
+        let session_builder: GenericSessionBuilder<DefaultMode> =
+            SessionBuilder::new().known_node(uri);
+        session_builder
+    }
+
+    #[cfg(feature = "cloud")]
+    {
+        use crate::CloudSessionBuilder;
+        use std::path::Path;
```

Comment:
Why are these imports here and the remaining ones are on the top of the file?

Reply:
The imports are only used in this block, the others on the top of the file allow to create the conditional type `ConditionalSessionBuilderKind`.